### ISSUE TITLE
crates.io version panics

### ISSUE THREAD

**Issue by tshepang** [assert-rs/assert_cli#115]
- `assert_cli` version: 0.6.1
- Rust version: rustc 1.28.0-nightly (cd494c1f0 2018-06-27)
- OS and version: Debian testing

```
   Assert::command(&["ls", "/"])
        .current_dir(repo)
        .stdout()
        .is("")
        .execute()?;
 ```
Above results in this panic:

> thread 'main' panicked at 'attempt to subtract with overflow', /home/tshepang/.cargo/registry/src/github.com-1ecc6299db9ec823/assert_cli-0.6.1/src/diff.rs:18:29

git master instead gives:

> Assertion failed for `ls /`

**Comment by epage** [comment]
Ok, so since master works, I'm assuming that is because of #115.

Would you be willing to give [`assert_cmd`](https://github.com/assert-rs/assert_cmd/) a try?   We're prepping it as the direct replacement for assert_cmd and we'll eventually transform assert_cli into something more.  It shouldn't have this problem and it'd be good to get more feedback on it.

I have [an example for cobalt converting](https://github.com/cobalt-org/cobalt.rs/pull/465).  The main thing missing atm is saying "why" things failed.  

If that doesn't work for you, we can go ahead and get another assert_cli release out.

**Comment by tshepang** [comment]
I think you meant to link to a different issue number?

**Comment by tshepang** [comment]
By "why", do you mean it does not do diffs (expected vs. got)?

**Comment by epage** [comment]
Fixed the PR number.

And yes, it doesn't do the diffs yet.

**Comment by tshepang** [comment]
ok, I do need those diffs. Without those, how do you know what failed?

**Comment by epage** [comment]
Yeah, its less than ideal.  You know what line in your code failed but you don't know why it failed.  It forces you to go and manually reproduce the test setup.

Since that feature is a blocker for you, I'll try to get a new assert_cli release out later today or tomorrow.

Hopefully it won't take too long for me to finish adding the explanation for why a test failed to assert_cmd so it can replace assert_cli for you.

**Comment by tshepang** [comment]
Ok, thanks in advance, and I sure will check assert_cmd soon
